src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -17,6 +17,10 @@ public void setElevatorPosition(double position) {
     io.setElevatorPosition(position);
   }
 
+  public void setElevatorSpeed(double speed) {
```

Comment:
Needs a docstring, make sure to specify what units speed is in and its range (e.g. 0 to 1)

Reply:
https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/controls/DutyCycleOut.html#withOutput(double)